src/Roslyn/Conventional.Roslyn/Conventions/SolutionDiagnosticAnalyzerConventionSpecification.cs

```diff
@@ -33,7 +33,8 @@ public IEnumerable<ConventionResult> IsSatisfiedBy(Solution solution)
         private IEnumerable<ConventionResult> IsSatisfiedBy(Document document)
         {
             var node = document.GetSyntaxRootAsync().Result;
-            if (document.TryGetSemanticModel(out var semanticModel))
+            var semanticModel = document.GetSemanticModelAsync().Result;
```

Comment:
Let's use `.GetAwaiter().GetResult()` here to ensure any exceptions are propagated: https://stackoverflow.com/a/38530225

Reply:
🤔 Could this also be applied to the `document.GetSyntaxRootAsync()` call as well?